source/client/index.js

```diff
@@ -1,13 +1,20 @@
 import React from 'react';
-import { render } from 'react-dom';
-import createApp from 'shared/components/app';
+import { renderToString } from 'react-dom/server';
+import { RoutingContext } from 'react-router';
 
-const App = createApp(React);
-const props = window.payload || {
-  title: 'Default client title'
+import { Provider } from 'react-redux';
+import configureStore from 'shared/configureStore';
+
+const store = configureStore(window.BOOTSTRAP_CLIENT_STATE);
+
+store.subscribe(() => console.log(store.getState()));
+
+const createDOM = renderProps => {
+  return renderToString(
+    <Provider store={store}>
+      <RoutingContext { ...renderProps } />
+    </Provider>
+  );
 };
 
-render(
-  <App { ...props } />,
-  document.getElementById('root')
-);
+export default createDOM;
```

Comment:
As far as I can see nothing is rendered here. The default export is a function returning the rendered page as a string. I read the redux-simple-router docs and thought something like this could work:
    import React from 'react';
    import ReactDOM from 'react-dom';
    import { Router } from 'react-router';
    import { Provider } from 'react-redux';
    import createBrowserHistory from 'history/lib/createBrowserHistory';
    import { syncReduxAndRouter } from 'redux-simple-router'
    import configureStore from 'shared/configureStore';
    import routes from 'shared/routes.js'

    const store = configureStore(window.BOOTSTRAP_CLIENT_STATE);
    const history = createBrowserHistory();
    store.subscribe(() => console.log(store.getState()));
    syncReduxAndRouter(history, store);

    let root = document.getElementById('root');
    ReactDOM.render(
    	<Provider store={store}>
    		<Router history={history} children={routes} />
    	</Provider>,
      root
    );